src/pages/clients-list/forms/ClientTemplateForm.js

```diff
@@ -1301,7 +1299,7 @@ const ClientTemplateForm = ({ setErrorMessage, recordId, _labels, plantId, maxAc
                         Component: AddressFormShell,
                         props: {
                           readOnly: editMode,
-                          required: true,
+                          optional: true,
```

Comment:
the address is required, the work address is optional.
the props naming or value is wrong here